src/pages/Home/index.tsx

```diff
@@ -1,12 +1,30 @@
-import Map from '../../components/common/Map';
+import { useDispatch } from 'react-redux';
+import { userIcon } from '../../assets';
+import { Button, Paragraph } from '../../components/common';
+import { logOut } from '../../state/actions/userActions';
+import useAuthentication from '../../hooks/useAuthentication';
 
 function Home() {
+  const dispatch = useDispatch();
+  const { username } = useAuthentication();
   return (
-    <section className="flex h-full">
-      {/* TODO: remove this empty div and create the component containing it's own logic */}
-      <div className="w-2/6 h-screen max-w-sm bg-white" />
-      <Map />
-    </section>
+    <>
+      <div>
+        <img className="mb-4" src={userIcon} alt="User profile" />
+        <Paragraph text={username} />
+        <Button
+          text="Logout"
```

Comment:
I recommend you to use i18n for all visible texts inside your apps. Even when you only support one language, it's always recommended to have internationalization inside your projects because it provides more maintainability for the future. 

It's only a suggestion, you don't need to change anything for this project.

Reply:
Completly agree with you Marcos!
I didn't add i18n to this project since Target is more like a internal project that isn't meant to be online, but in a real case scenario I'd have internationalization applied. 
Thank you for the review <3